### ISSUE TITLE
Expose hyper-parameter settings for deep-learning scenarios in cli

### ISSUE THREAD

**Issue by LittleLittleCloud** [dotnet/machinelearning-modelbuilder#2526]
## Text-Classification
- epoch
- batchsize
- architecture
- device

## Sentence-Similarity
- epoch
- batchsize
- architecture
- device

## Image Classification
- Arch
- batchsize
- epoch
- device

**Comment by luisquintanilla** [comment]
Is Arch in image classification which base model to use?

**Comment by LittleLittleCloud** [comment]
@luisquintanilla Yep, will be one of `resnet101`, `resnet50`, `resnet18`, `alexnet`

**Comment by zewditu** [comment]
@LittleLittleCloud could you have some explanation of what value will be added by exposing hyper-parameter settings to the user?

**Comment by luisquintanilla** [comment]
@LittleLittleCloud are we okay to close this one? Did this make it into the latest release?
